CMakeLists.txt

```diff
@@ -2354,6 +2364,7 @@ if (is_multiconfig)
   group_sources(src)
   group_sources(docs)
   group_sources(Builds)
+  group_sources(.nih_c)
```

Comment:
I think you might want to `group_sources(${nih_cache_path})` instead, which gives just the cache portion for the relevant compiler/generator combo. That said, on xcode this ends-up only including the files which were gotten via FetchContent on the first run (when this cache is clean). On subsequent runs, it's very likely this will contain the other ExternalProject files as well - just something to be aware of I think.

Reply:
I tried this (and a couple of relative path variations). Because of the way that `group_sources` is implemented, it doesn't work as-is. Also, without it, if you've got multiple config types (eg MSVC and ninja) then the _other_ files will still be at the top of the tree (the files stuck at the top of the tree is why I added this in the first place).

Now the bigger issue is that the only reason (as far as I can tell) that `.nih_c` is relevant is because we search for _all_ `.md` files in the repo, and there are several under `.nih_c`. I think the better way to do this is to only do `Builds/*.md docs/*.md src/*.md` (which will still be relevant accounting for your suggestion below).